src/SFA.DAS.LoginService.Configuration/SFA.DAS.LoginService.Configuration.csproj

```diff
@@ -5,7 +5,8 @@
     </PropertyGroup>
 
     <ItemGroup>
-      <PackageReference Include="Newtonsoft.Json" Version="12.0.1" />
+      <PackageReference Include="Newtonsoft.Json" Version="11.0.2" />
```

Comment:
Annoying Newtonsoft horrible dependecy problems

Reply:
yes, I'm sure it doesn't matter but the DAS NSB package insists, and I don't know why